### DISCUSSION TITLE
Free servers in the CIS, Adding P2P broadcasting of screen demonstrations and videos

### DISCUSSION THREAD

**Discussion by strobecsmain** [stoatchat/discussions#838] [Feature Suggestions]
Hello to the friends and team at Revolt! My name is Arseniy, and I would like to help reduce the load on Revolt's infrastructure in Russia and Belarus due to recent events in Russia, specifically the blocking of Discord. My company is decommissioning some "old" server equipment, and I plan to retrieve it and make it fully available under your control.

Additionally, I would like to request the implementation of peer-to-peer (P2P) broadcasting for audio and video based on the QUIC or DASH protocols, which cannot be blocked via DPI in Russia. I also would like to offer my help with localization efforts for the Russian and Ukrainian languages.

Let me know how I can assist further.

**Comment by baglayan** [comment]
For localization, check out https://translate.revolt.chat/projects/revolt/
